README.md

```diff
@@ -1 +1,268 @@
-# afm-helpers
\ No newline at end of file
+# afm-helpers
+
+## Current
+
+```javascript
+expect(
+  MeasureBuilder("local_identifier")
+    .withSimpleMeasureDefinition("gd_identifier")
+    .build()
+).toEqual(
+  {
+    localIdentifier: "local_identifier",
+    definition: {
+      measure: {
+        item: {
+          identifier: "gd_identifier"
+        }
+      }
+    }
+  }
+)
+```
+
+## Proposed
+
+* Consider renaming to gdc-afm-helpers to comply with [gdc-afm-connect](https://www.npmjs.com/package/@gooddata/gdc-afm-connect)
+* Consider using TypeScript
+* Automatically distinguish between uri and identifier
+* Generate localIdentifiers automatically if not supplied
+
+> Do not confuse `<Visualization measures={…} />` with `execution.afm.measures`!
+
+Q: Is it planned to have builders for both afm and bucket-interface as well?
+
+### Measures
+
+#### Simple measure using uri
+
+```javascript
+expect(
+  AFMBuilder.measure('/gdc/md/abcd/obj/123')
```

Comment:
The proposed API looks nice! The only thing I don't really know how to solve is differentiating between all the measure types. Simple `.measure(id)` is not enough, imo. Am I missing something here?

Reply:
I implemented the proposed API in a feature branch in my project: https://github.com/no23reason/gdc-afm-helpers/blob/measure-proposal/src/builders/AFMBuilder.ts
With test of your most complex example:
https://github.com/no23reason/gdc-afm-helpers/blob/measure-proposal/src/builders/__tests__/AFMBuilder.test.ts

 Other than the comment above it matches your proposal. Could you please check it out?